masu/processor/_tasks/process.py

```diff
@@ -45,17 +45,19 @@ def _process_report_file(schema_name, report_path, compression):
                                 report_path,
                                 compression)
     LOG.info(log_statement)
-
     file_name = report_path.split('/')[-1]
+
     stats_recorder = ReportStatsDBAccessor(file_name)
+    stats_recorder.log_last_started_datetime()
+    stats_recorder.commit()
```

Comment:
@adberglund Need your db knowledge here.  Is there any unintended consequences of committing here, updating, and committing again later while maintaining the session created in`ReportStatsDBAccessor`?

Reply:
Pretty certain this is fine. Since the log is the only "dirty" object on the session at commit time, it gets committed and that particular transaction is complete. You can then re-use the session to do new stuff (I'm not certain but it might actually retire the session, throw it back in a pool, and then re-grab it again when you call it again), and commit again when done. SQLAlchemy's session will mark dirty state again and consider it a new transaction.